ui/app/templates/components/wizard/kv-engine.hbs

```diff
@@ -5,6 +5,6 @@
   @docPath="/docs/secrets/kv/index.html"
 >
   <p>
-    The kv secrets engine is used to store arbitrary secrets within the configured physical storage for Vault. This backend can be run in one of two modes. It can be a generic Key-Value store that stores one value for a key. Versioning can be enabled and a configurable number of versions for each key will be stored.
+    The KV Secrets Engine is used to store arbitrary secrets within the configured physical storage for Vault. This backend can be run in one of two modes. It can be a generic Key-Value store that stores one value for a key. Versioning can be enabled and a configurable number of versions for each key will be stored.
```

Comment:
```suggestion
    The KV Secrets Engine is used to store arbitrary secrets within the configured physical storage for Vault. This backend can be run in one of two modes. It can be a generic key-value store that stores one value for a key. Versioning can be enabled and a configurable number of versions for each key will be stored.
```